django_mongodb/manager.py

```diff
@@ -0,0 +1,7 @@
+from django.db.models.manager import BaseManager
+from .query import MongoQuerySet
+
+
+class MongoManager(BaseManager.from_queryset(MongoQuerySet)):
+    def get_query_set(self):
```

Comment:
This should be `get_queryset`.

Reply:
I took it out because as far as I can tell we don't need to customize.